vms/platformvm/block/executor/acceptor.go

```diff
@@ -116,46 +116,14 @@ func (a *acceptor) ApricotAtomicBlock(b *block.ApricotAtomicBlock) error {
 	return nil
 }
 
-func (a *acceptor) abortBlock(b block.Block, blockType string) error {
+func (a *acceptor) optionBlock(b block.Block, blockType string) error {
 	parentID := b.Parent()
 	parentState, ok := a.blkIDToState[parentID]
 	if !ok {
 		return fmt.Errorf("%w: %s", state.ErrMissingParentState, parentID)
 	}
 
-	if a.bootstrapped.Get() {
-		if parentState.initiallyPreferCommit {
-			a.metrics.MarkOptionVoteLost()
-		} else {
-			a.metrics.MarkOptionVoteWon()
-		}
```

Comment:
We don't set this value during execution, so the acceptor doesn't really have access to it anymore.